src/orderly/current.py

```diff
@@ -63,15 +67,17 @@ def interactive():
             name=path.name,
             id=None,
             orderly=OrderlyCustomMetadata(),
+            # TODO: expose a way of configuring this
```

Comment:
The way this is done from R is pretty gross, tbh, so this might be worth discussing generally at some point